RecoJets/JetProducers/plugins/VirtualJetProducer.cc

```diff
@@ -228,8 +228,6 @@ VirtualJetProducer::VirtualJetProducer(const edm::ParameterSet& iConfig) {
 		fjRangeDef_ = RangeDefPtr( new fastjet::RangeDefinition(rhoEtaMax) );
 	} 
 
-	if( doFastJetNonUniform_ ) puCenters_ = iConfig.getParameter<vector<double> >("puCenters"); 	/// where is this used? 
```

Comment:
This is used, for example, at lines 566 and 578 of this file: you can't get rid of it as such!
(Probably the comment in the original code wanted to mean "Who's using the flag doFastJetNonUniform_?". But I'm just guessing...